pallets/registrar/src/lib.rs

```diff
@@ -595,14 +627,17 @@ pub mod pallet {
     }
 
     impl<T: Config> Pallet<T> {
-        pub fn is_para_manager(para_id: &ParaId, account: &T::AccountId) -> bool {
+        pub fn is_para_manager(para_id: &ParaId, account: T::AccountId) -> bool {
             // This check will only pass if both are true:
             // * The para_id has a deposit in pallet_registrar
-            // * The deposit creator is the signed_account
-            RegistrarDeposit::<T>::get(para_id)
+            // * The signed_account is either the deposit creator or the para manager
+            let is_creator = RegistrarDeposit::<T>::get(para_id)
                 .map(|deposit_info| deposit_info.creator)
                 .as_ref()
-                == Some(account)
+                == Some(&account);
+
+            // Short circuit to avoid a DB read if is_creator
+            is_creator || ParaManager::<T>::get(para_id) == Some(account)
```

Comment:
This works, but not sure if we want to have more than 1 manager account. I imagine `set_para_manager` will transfer ownership, but in reality it will add an additional manager.

So I would do something like:
```suggestion
            ParaManager::<T>::get(para_id).unwrap_or(deposit_info.creator) == Some(account)
```

Reply:
As you mentioned, I approached this as being able to add an additional manager, thinking that most of the time the creator account will be the only manager. I can change it so there's only one manager, and on registration `ParaManager` is set to the creator and `is_para_manager` will then only check `ParaManager`. What do you think?
(c.c @girazoki @ParthDesai)

Reply:
I think what we want to achieve is to have an additional manager that is what we call "`administrator`". This is the go-to approach in pallet-assets.

Reply:
I like `ParaAdministrator` more than `ParaManager`, to not confuse with the already established manager term in the pallet. Will change it

Reply:
It was decided that the account stored in `ParaManager` will be the only manager. The PR has been updated so the parachain creator (and only the creator) can at any time set a new manager